browser/sync/BUILD.gn

```diff
@@ -10,3 +10,25 @@ source_set("sync") {
     "//components/sync_device_info",
   ]
 }
+
+source_set("unit_tests") {
+  testonly = true
+
+  if (is_ios) {
+    # Chromium/iOS shouldn't use anything in //chrome, so disable these tests
+    # for iOS
+    sources = []
```

Comment:
let's keep it as usual in this form:
```
sources = []
deps = []
if (!is_ios) {
  sources += [...]
  deps += [...]
}
```

Reply:
thanks, fixed at 
https://github.com/brave/brave-core/pull/14402/commits/7b7aaad7ec1042ba7c0c9e8e74ca9c57e89debbb